examples/demo-rollup/src/celestia_rollup.rs

```diff
@@ -43,54 +67,24 @@ impl RollupTemplate for CelestiaDemoRollup {
         .await
     }
 
-    fn create_vm(&self) -> Self::Vm {
-        Risc0Host::new(risc0::ROLLUP_ELF)
-    }
-
-    fn create_verifier(&self) -> <Self::DaService as DaService>::Verifier {
-        CelestiaVerifier {
-            rollup_namespace: ROLLUP_NAMESPACE,
-        }
-    }
-
-    fn create_zk_storage(
```

Comment:
Can we keep the `ZkStorage` here and pass it to the `STFRunner` as a parameter? The runner needs it only for proving and this functionality will be encapsulated by a `ProverService` trait.

Reply:
Updated